### ISSUE TITLE
Analytics Standards for Staging sites?

### ISSUE THREAD

**Issue by geramirez** [18F/analytics-standards#3]

**Comment by afeld** [comment]
Curious about this as well. For [C2](https://github.com/18F/C2/), for example, we have local dev, staging, and production server environments, and I'm not sure whether the analytics should be enabled on all of them or not. Seems like they could be distinguished on the GA end with a hostname filter? Not sure if there are other downsides though. More info: 
- http://programmers.stackexchange.com/questions/139603/google-analytics-with-multiple-environments
- http://gatipoftheday.com/apply-a-hostname-filter-for-development-and-staging-servers/

**Comment by gbinal** [comment]
Here's what I'd say:  
- I don't think there's any obligation to have analytics on the staging sites, so it's okay to only have it on production.  
- Often, though, the same code will be maintained for production and staging and so it's convenient to have both feeding into the same GA.  This is okay.  It's easy to filter the difference.  
- If this is a concern, it is easy to set up filters to separate production and staging and/or another view that shows the full URL in GA so that the pages can be easily distinguished.  

To summarize though, it's not a problem and there's not currently a standard for how a project should handle this.  But if they have questions or need help, they should come to #g-analytics.

**Comment by konklone** [comment]
Note that this is an ongoing item with DAP - how to publish the list of all participating domains and subdomains, in a way that doesn't require manual work to exclude staging/dev/etc. 

So: if you use DAP on staging/dev, be prepared for the possibility that staging/dev domains may become part of public datasets.

(Also: it's not impossible that they become part of a public dataset, whether or not you use DAP.)
